internal/services/cdn/cdn_frontdoor_endpoint_resource.go

```diff
@@ -117,7 +118,7 @@ func resourceCdnFrontDoorEndpointRead(d *pluginsdk.ResourceData, meta interface{
 	ctx, cancel := timeouts.ForRead(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
-	id, err := parse.FrontDoorEndpointID(d.Id())
+	id, err := parse.FrontDoorEndpointIDInsensitively(d.Id())
```

Comment:
this is coming from the config and so should be case-sensitive

```suggestion
	id, err := parse.FrontDoorEndpointID(d.Id())
```